emily/handler/src/database/entries/mod.rs

```diff
@@ -246,10 +246,20 @@ pub(crate) trait TableIndexTrait {
         // Convert inputs into the types needed for querying.
         let exclusive_start_key =
             maybe_exclusive_start_key_from_next_token::<Self::SearchToken>(maybe_next_token)?;
+
+        let table_name = Self::table_name(settings);
+        let index_name = format!("{:#?}", Self::INDEX_NAME_IF_GSI);
+        let partition_key_name = <<Self::Entry as EntryTrait>::Key as KeyTrait>::PARTITION_KEY_NAME;
+        tracing::debug!(
+            %table_name,
+            %index_name,
+            %partition_key_name,
+            "querying dynamoDB",
+        );
```

Comment:
Oh I think my previous suggestion was a little off.
```suggestion
        let index_name = Self::INDEX_NAME_IF_GSI;
        let partition_key_name = <<Self::Entry as EntryTrait>::Key as KeyTrait>::PARTITION_KEY_NAME;
        tracing::debug!(
            %table_name,
            ?index_name,
            %partition_key_name,
            "querying dynamoDB",
        );
```

Reply:
Yeah, this way it works.. [10819c4](https://github.com/stacks-sbtc/sbtc/pull/1625/commits/10819c4abb3f050b9c30c36bb48c9f580c0aecea)